src/ukrdc_cupid/core/store/delete_old.py

```diff
@@ -0,0 +1,132 @@
+import ukrdc_sqla.ukrdc as sqla
```

Comment:
@George-D-S  ignore anything with old in front of it. I will clean these up at some point but for now some of these files have stuff I need in.